ledger/sandbox-common/src/test/lib/scala/platform/sandbox/cli/CommonCliSpecBase.scala

```diff
@@ -268,8 +268,8 @@ abstract class CommonCliSpecBase(
 
     "parse the metrics reporting interval when given" in {
       checkOption(
-        Array("--metrics-reporting-interval", "PT1M30S"),
-        _.copy(metricsReportingInterval = Duration.ofSeconds(90)),
+        Array("--metrics-reporting-interval", "1.5m"),
```

Comment:
@cocreature Is this cli option change OK? This is due to the change from Java Duration to scala duration

Reply:
This seems like a breaking change. Can we continue accepting a Java duration here? Or alternatively accept both preferring a Java duration. We can still convert to a Scala duration at the very beginning if that helps.

Reply:
You should be able to use a custom reader that tries one first and if that fails tries the other.

Reply:
Yeah just wrote one, I will make another PR as I forgot to remove the label here.